lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -3130,8 +3120,16 @@ Optional<ActorIsolation> ConformanceChecker::checkActorIsolation(
         witness->getAttrs().hasAttribute<NonisolatedAttr>())
       return None;
 
+    //check that the results of the witnessing method are sendable
+    diagnoseNonSendableTypesInReference(
+        getDeclRefInContext(witness), DC, loc, SendableCheckReason::Conformance,
+        None, FunctionCheckKind::Results);
+
+    //check that the parameters of the requiring method are sendable
```

Comment:
- generally we refer to it as "method requirement"
- I suggest putting at least a space between `//` and the start of a comment, with capitalization of proper nouns like Sendable as additionally appreciated.

Reply:
comment spacing and capitalization fixed! don't totally understand what phrase you wanted me to replace with "method requirement" though

Reply:
ah I meant `requiring method` -> `method requirement` but it's minor